Elizabeth_Theme/src/css/_colors.scss

```diff
@@ -0,0 +1,2 @@
+$theme-main-emphasize-text: #D6A261;
+$theme-main-buttons: #D6A261;
```

Comment:
These are the same CSS color values.  Should we just make them 1 variable instead of 2?